po/de.po

```diff
@@ -1023,44 +1042,42 @@ msgid ""
 "@to_pay on @money, every task tagged with @to_pay will also appear when you "
 "select @money."
 msgstr ""
-"Um Ihre Aufgaben zu organisieren nutzen Sie in GTG Markierungen.  Eine "
-"Markierung ist ein einfaches Wort, dass mit &quot;@&quot; beginnt.\n"
+"Um Ihre Aufgaben zu organisieren nutzen Sie in GTG Schlagwörter. Ein "
+"Schlagwort ist ein einfaches Wort, dass mit &quot;@&quot; beginnt.\n"
 "\n"
-"Versuchen Sie, ein mit &quot;@&quot; beginnendes Wort zu schreiben.\n"
+"Versuchen Sie, hier ein mit &quot;@&quot; beginnendes Wort zu schreiben:\n"
 "\n"
-"Sobald es gelb erscheint, ist es eine Markierung.  Und diese Markierung ist "
-"nun mit der Aufgabe verknüpft!\n"
+"Sobald es gelb erscheint, ist es dann ein Schlagwort, welcher mit der "
+"Aufgabe verknüpft ist.\n"
 "\n"
-"Mit dem Menüeintrag Ansicht, können Sie eine Ansicht öffnen die alle "
-"benutzten Markierungen anzeigt.  Damit können alle mit einer Markierung "
-"verknüpften Aufgaben sichtbar gemacht werden.\n"
+"Über den Menüknopf oder der F9-Taste kann man die Seitenleiste ein- oder "
+"ausblenden, welche alle verwendeten Schlagwörter zeigt. Damit können alle "
+"mit einem Schlagwort verknüpften Aufgaben angezeigt werden.\n"
 "\n"
-"Mit einem Rechtsklick auf einer Markierung kann man diese auch bearbeiten.  "
-"Damit lassen sich beispielsweise eine Farbe oder ein Symbol verbinden.  Dies "
-"ist ein Vorteil wenn Sie rasch die Aufgaben Ihrer Aufgabenliste über die mit "
-"ihnen verknüpften Markierungen identifizieren wollen.\n"
+"Mit einem Rechtsklick auf einem Schlagwort in der Seitenleiste kann man "
+"diese auch bearbeiten. Damit lassen sich beispielsweise eine Farbe oder ein "
+"Symbol verbinden. Dies ist ein Vorteil, wenn Sie schnell die Aufgaben Ihrer "
+"Aufgabenliste über die mit ihnen verknüpften Schlagworte identifizieren "
+"wollen.\n"
 "\n"
-"Neue Markierungen werden stets ausschliesslich mit der gerade bearbeiteten "
-"Aufgabe verknüpft, und nicht einer untergeordneten Aufgabe.  Eine neue "
-"untergeordnete Aufgabe wird jedoch die Markierungen ihrer Hauptaufgabe "
-"übernehmen.\n"
+"Neue Schlagwörter werden stets ausschließlich mit der gerade bearbeiteten "
+"Aufgabe verknüpft, und nicht einer Teilaufgabe. Eine neue Teilaufgabe wird "
+"jedoch die Markierungen ihrer Übergeordnete Aufgabe übernehmen.\n"
```

Comment:
I'm not sure about this, as it might intentionally refer to the similarly named button, but normally "übergeordneten" instead of "Übergeordnete" would be correct.

Reply:
It doesn't really refer to the button, so I'm using your translation.
Also changed Markierungen → Schlagwörter here (sometimes Markierungen was used in the old translation, I tried to be more consistent about that)

Reply:
Perfect!